README.md

```diff
@@ -46,6 +46,9 @@ It is designed to offer pre-configured testing setups for many different types a
 - **SQL Server**: Version 2022 is available
 - **Spanner**: The latest cloud-emulator from Google is available
 - **Cockroach**: Version 23.1-latest is available
+- **Redis**: Latest server
+- **Dragonfly**: Latest server
+- **KeyDB**: Latest server
```

Comment:
Would Redis and it's derivatives be more fitting or no? idk if there are core differences there.

Reply:
No issues re-wording it, but derivatives may not be quite the right way to think about it.  I think Dragonfly is Redis compatible but isn't a "fork".  

Also, there are more advanced features that may only be implemented by each vendor.